doc/connectivity/bluetooth/overview.rst

```diff
@@ -93,3 +101,23 @@ Bluetooth stack.
     * Local controller support as a virtual HCI driver
 
   * Verified with multiple popular controllers
+
+* LE Audio in Host and Controller
+
+  * Full support for Isochronous channels
```

Comment:
Controller only has limited support for Isochronous channels so far - Host can be considered feature-complete, and only lacks from test commands implemented in https://github.com/zephyrproject-rtos/zephyr/pull/53945